docs/source/data_extraction.rst

```diff
@@ -10,8 +10,9 @@ Columns correspond to data about the material histories, such as the temperature
 Defining row and columns
 ------------------------
 
-A Row object describes a mapping from a list of datasets to rows of a table.
-The only such mapping presently supported is :class:`~citrine.ara.rows.MaterialRunByTemplate`, which produces one row per Material Run associated with any of a list of material templates.
+A Row object describes a mapping from a list of datasets to rows of a table by selecting a set of Material Histories.
+Each Material History corresponds to exactly one row, though the Material Histories may overlap such that the same objects contribute data to multiple rows.
```

Comment:
```suggestion
Each Material History corresponds to exactly one row, though the Material Histories may overlap such that the same objects contribute data to multiple rows.
For example, the material histories of distinct Material Runs will map to exactly two rows even if their histories are identical up to the Process Run of their final baking step.
```

Reply:
Process Run only has a single output_material, so this example is not possible.

Reply:
" up to the Process Run of their final baking step" i.e. the Process Runs differ.

Reply:
Updated suggestion to be clearer.